File: Views/src/components/financial/components/daily/in/dto/daily-in-flow-dto.ts

import { ClientDto } from "src/components/client/dto/client-dto";
import { PartnerDto } from "src/components/partner/dto/partner-dto";
import { TypePaymentDto } from "../../../../dto/type-payment-dto";


export class DailyInFlowDto {
  today: string;
  clientId: number;
  typepaymentId: number;
  checkingaccount: string;
  amount: number;
  description: string;
}

File: Services/Dto/Financial/DailyInFlowDto.cs
using System;
using System.Collections.Generic;
using Domain.Entities;

namespace Services.Dto.Financial
{
    public class DailyInFlowDto
    {

        public string Today { get; set; }
        public int ClientId { get; set; }
        public ClientDto Client { get; set; }
        public int TypepaymentId { get; set; }
        public TypePaymentDto Typepayment { get; set; }
        public int CheckingaccountId { get; set; }
        public CheckingAccountDto Checkingaccount { get; set; }
        public string Amount { get; set; }
        public string Description { get; set; }

    }
}